node/localnode.go

```diff
@@ -64,6 +64,7 @@ func (localNode *LocalNode) MarshalJSON() ([]byte, error) {
 	out["height"] = localNode.GetHeight()
 	out["uptime"] = time.Since(localNode.startTime).Truncate(time.Second).Seconds()
 	out["version"] = config.Version
+	out["beneficiaryAddr"] = config.Parameters.BeneficiaryAddr
```

Comment:
Also @gdmmx
Should we include beneficiaryAddr in public API? This may be a private info. If we just want to use it for dashboard, we can get it directly from local node

Reply:
get it.

Reply:
Agree. beneficiaryAddr should not be showed without authority.